### DISCUSSION TITLE
Can I get data in 3d space?

### DISCUSSION THREAD

**Discussion by jacktang** [NatLabRockies/floris#671] [Q&A] [answered]
Hello, 

I am learning floris from the examples and thanks for the great library! From `02_visualizations.py` I learnt that the wake can be plot from different viewport: horizontal, y and cross. And my question is can I get the wake data in 3d space? Thanks!

**Comment by paulf81** [comment, marked as answer]
Have a look at example examples/28_extract_wind_speed_at_points.p, which illustrates sampling the velocity at arbitrary points in 3d space

**Reply by jacktang** [reply]
Thanks for the example @paulf81 ! I will try to understand it and try to plot in 3d
